client/src/helpers/ManageBookings/index.ts

```diff
@@ -0,0 +1,5 @@
+import checkIfBooked from './checkIfBooked';
+import formatBookingDate from './formatBookingDate';
+import sortBookingsByStartDate from './sortBookingsByStartDate';
+
+export { checkIfBooked, formatBookingDate, sortBookingsByStartDate };
```

Comment:
It's nice to see consistent hierarchy here 👍🏼 However I think it's a bit over the top to separate each helper function into its own file. A single file can definitely qualify to contain more than a single function so I would recommend to group these together in a single file instead. Granted, this is a bit preferential but it will definitely reduce overhead in managing the project 🙌🏼

Reply:
I refactored all these separate files into named exports in /helpers/ManageBookings.tsx.